runners/spark/pom.xml

```diff
@@ -79,6 +79,8 @@
                     org.apache.beam.sdk.testing.UsesSplittableParDo,
                     org.apache.beam.sdk.testing.UsesCommittedMetrics
                   </excludedGroups>
+                  <parallel>none</parallel>
+                  <threadCount>4</threadCount>
```

Comment:
It apparently has.. without this the Metrics container in the Spark runner (which is cleared before each test) is being used by another execution and throws NPE.